docs/adr/2021-08-26-code-of-conduct.md

```diff
@@ -15,7 +15,6 @@ We want to make people feel welcome, and so should have a formal CoC.
 ## Decision Drivers
```

Comment:
We should remove this file from this pr and leave intact. Architectural decision records can't be modified.

Reply:
Okay